testing/src/main/java/org/oppia/testing/TestLogReportingModule.kt

```diff
@@ -1,23 +1,15 @@
 package org.oppia.testing
 
+import dagger.Binds
 import dagger.Module
-import dagger.Provides
-import java.lang.Exception
-import javax.inject.Singleton
-import org.oppia.util.firebase.CrashLogger
+import org.oppia.util.logging.ExceptionLogger
 
 /**
  * Provides fake crash logging dependencies.
  */
 @Module
-class TestLogReportingModule {
-  companion object {
-    var exceptionList = ArrayList<Exception>()
-  }
+abstract class TestLogReportingModule {
 
-  @Provides
-  @Singleton
-  fun provideCrashlyticsLogger(): CrashLogger {
-    return FakeCrashLogger()
-  }
+  @Binds
+  abstract fun bindFakeCrashLogger(fakeExceptionLogger: FakeExceptionLogger): ExceptionLogger
```

Comment:
Optional, but I think you could make the class an interface instead since it only contains abstract methods.

Reply:
Done.